packages/otelbin/src/components/monaco-editor/ValidationErrorConsole.tsx

```diff
@@ -42,58 +51,123 @@ export default function ValidationErrorConsole({ errors, font }: { errors?: IErr
 		}
 	}, [errorCount, warningsCount]);
 
+	const onHeightChange = useCallback((newHeight: number) => {
+		setHeight(newHeight);
+	}, []);
+
+	const state = useRef<State>({
+		dragging: false,
+		initialY: 0,
+		initialHeight: 0,
+	});
+
+	useEffect(() => {
+		const onMouseUp = (e: MouseEvent) => {
+			e.preventDefault();
+			e.stopPropagation();
+			state.current.dragging = false;
+		};
+
+		const onMouseMove = (e: MouseEvent) => {
+			e.stopPropagation();
+			const minHeight = 300;
+			const maxHeight = 850;
+			const { dragging, initialY, initialHeight } = state.current;
+
+			if (dragging && e.clientY < maxHeight && e.clientY > minHeight) {
+				const deltaY = initialY - e.clientY;
+				const newHeight = initialHeight + deltaY;
+				onHeightChange(newHeight);
+			}
+		};
+
+		window.addEventListener("mouseup", onMouseUp);
+		window.addEventListener("mousemove", onMouseMove);
+
+		return () => {
+			window.removeEventListener("mouseup", onMouseUp);
+			window.removeEventListener("mousemove", onMouseMove);
+		};
+	}, [state, onHeightChange]);
+
 	return (
-		<div
-			className={`absolute bottom-0 left-0 z-10 ${
-				isOpenErrorConsole ? "h-[120px]" : "h-[37px]"
-			} w-full border-t-1 border-subtle bg-default pb-1 pt-1 transition-all`}
-		>
-			<div className="flex flex-col h-full">
-				<div className="flex items-center">
-					<ErrorAndWarningCounter
-						errorsCount={errorCount}
-						warningsCount={0}
-						isOpen={isOpenErrorConsole}
-						setOpen={setIsOpenErrorConsole}
-					/>
-					<ErrorAndWarningCounter
-						isWarning
-						errorsCount={0}
-						warningsCount={warningsCount}
-						isOpen={isOpenErrorConsole}
-						setOpen={setIsOpenErrorConsole}
-					/>
-				</div>
-				{isOpenErrorConsole && (
-					<div className="mt-2 flex h-[calc(100%-45px)] flex-col gap-y-1 overflow-auto px-[25px]">
-						{errors?.customWarnings &&
-							errors.customWarnings?.length > 0 &&
-							errors.customWarnings.map((warning: string, index: number) => {
-								return <ErrorMessage key={index} customWarnings={warning} font={font} />;
-							})}
-						{serverSideValidationResult.result?.error && (
-							<ErrorMessage
-								serverSideError={
-									serverSideValidationResult.result?.message + " - " + serverSideValidationResult.result?.error
-								}
-								font={font}
-							/>
-						)}
-						{errors?.ajvErrors &&
-							errors.ajvErrors?.length > 0 &&
-							errors.ajvErrors.map((error: IAjvError, index: number) => {
-								return <ErrorMessage key={index} ajvError={error} font={font} />;
-							})}
-						{errors?.customErrors &&
-							errors.customErrors?.length > 0 &&
-							errors.customErrors.map((error: string, index: number) => {
-								return <ErrorMessage key={index} customErrors={error} font={font} />;
-							})}
-						{errors?.jsYamlError?.mark?.line && <ErrorMessage jsYamlError={errors?.jsYamlError} font={font} />}
-					</div>
-				)}
+		<>
```

Comment:
**Question:** Why do we need this fragment?

Reply:
fixed and pushed. @bripkens